test/ts/logs/collateralized_contract.ts

```diff
@@ -58,3 +75,41 @@ export function CollateralSeized(
         name: "CollateralSeized",
     };
 }
+
+export function ERC721CollateralReturned(
+    contract: Address,
+    agreementID: Bytes32,
+    debtor: Address,
+    erc721Contract: Address,
+    tokenId: UInt,
+): ABIDecoder.DecodedLog {
+    return {
+        address: contract,
+        events: LogUtils.getParams([
+            ["agreementID", agreementID],
+            ["debtor", debtor],
+            ["erc721Contract", erc721Contract],
+            ["tokenID", tokenId],
+        ]),
+        name: "CollateralReturned",
+    };
+}
+
+export function ERC721CollateralSeized(
+    contract: Address,
+    agreementID: Bytes32,
+    beneficiary: Address,
+    erc721Contract: Address,
+    tokenId: UInt,
+): ABIDecoder.DecodedLog {
+    return {
+        address: contract,
+        events: LogUtils.getParams([
```

Comment:
so i was hoping to deprecate `getParams` in favor of `generateParam`. would love to not add any more instances of `getParams`.